composer/src/composer.rs

```diff
@@ -694,6 +707,12 @@ impl Builder {
         Ok(self)
     }
 
+    /// set the network mode for this test.
+    pub fn network_mode(mut self, mode: NetworkMode) -> Result<Builder, Error> {
```

Comment:
Just noticed this, why the Error?

Reply:
eh, good sighting. It came with copying from existing functions. Will return just `Builder`

Reply:
fixed now.